src/core/level.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * This module updates the title and shortName
+ * of a spec when there is a valid level configuration.
+ *
+ * Levels should be integers >= 0.
+ */
+
+import { getIntlData, showInlineWarning } from "./utils.js";
+export const name = "core/level";
+
+const localizationStrings = {
+  en: {
+    level: "Level",
+  },
+};
+
+const l10n = getIntlData(localizationStrings);
+
+export function run(conf) {
+  const h1Elem = document.querySelector("h1#title");
+
+  if (conf.hasOwnProperty("level")) {
+    if (Number.isInteger(conf.level) && conf.level >= 0) {
```

Comment:
Please see Slack for why I think we should use `const level = parseInt(conf.level);`